build.fsx

```diff
@@ -203,22 +261,22 @@ Target.create "All" ignore
 Target.create "PackAll" ignore
 
 "Clean"
-    ==> "Restore"
-    ==> "Build"
-    ==> "RunUnitTests"
-    ==> "StartStarWarsServer"
-    ==> "StartIntegrationServer"
-    ==> "RunIntegrationTests"
-    ==> "All"
-    =?> ("GenerateDocs", Environment.environVar "APPVEYOR" = "True")
+==> "Restore"
+==> "Build"
+==> "RunUnitTests"
+==> "StartStarWarsServer"
+==> "StartIntegrationServer"
+==> "RunIntegrationTests"
+==> "All"
+=?> ("GenerateDocs", Environment.environVar "APPVEYOR" = "True")
 
 "CleanDocs" ==> "GenerateDocs"
 
 "PackShared"
-    ==> "PackServer"
-    ==> "PackClient"
-    ==> "PackMiddleware"
-    ==> "PackAll"
+==> "PackServer"
+==> "PackClient"
+==> "PackMiddleware"
+==> "PackAll"
 
 Target.runOrDefaultWithArguments "All"
 
```

Comment:
I also tried to format it with Fantomas and it looks not so much readable. @ivelten I should have exclude it from formatting.
Could you to that and only apply your additions

Reply:
Sure, I will do that.